pkg/utils/openshift/utils.go

```diff
@@ -15,10 +15,23 @@ func IsOpenShift(cfg *rest.Config) (bool, error) {
 
 /*
 GetPlatformInfo examines the Kubernetes-based environment and determines the running platform, version, & OS.
+Returned PlatformInfo.OCPVersion is derived from underlying PlatformInfo.K8SVersion.
 Accepts <nil> or instantiated 'cfg' rest config parameter.
 
-Result: PlatformInfo{ Name: OpenShift, OCPVersion: 4.1.0, K8SVersion: 1.13+, OS: linux/amd64 }
+Result: PlatformInfo{ Name: OpenShift, OCPVersion: 4.1, K8SVersion: 1.13+, OS: linux/amd64 }
 */
 func GetPlatformInfo(cfg *rest.Config) (platform.PlatformInfo, error) {
```

Comment:
Again, assuming I understand correctly, the PlatformInfo "pretends" to contain the OpenShift version, but does not. In fact it requires a call to another function to either map a known version, or properly retrieve it.

If my understanding above is correct, I propose creating a new (exported/public) struct for OpenShift version and taking that info out of PlatformInfo. Then have those two functions just return that struct.